pkg/cache/cluster.go

```diff
@@ -633,30 +636,17 @@ func (c *clusterCache) watchEvents(ctx context.Context, api kube.APIResourceInfo
 			return err
 		}
 
-		defer func() {
-			w.Stop()
-			resourceVersion = ""
-		}()
-
-		var watchResyncTimeoutCh <-chan time.Time
-		if c.watchResyncTimeout > 0 {
```

Comment:
Need to update the docstring for `watchResyncTimeout` to indicate that we're not re-listing if we have a resourceVersion, we're just re-establishing watches.

Reply:
Done.